external/source/exploits/cve-2015-0057/cve-2015-0057/cve-2015-0057.c

```diff
@@ -0,0 +1,519 @@
+/*
+ * MS15-010 / CVE-2015-0057 win32k!xxxEnableWndSBArrows Use After Free
+ *
+ * Authors:
+ *   Udi Yavo (Vulnerability discover and details)
+ *   Spencer McIntyre (Exploit)
+ *
+ * References:
+ *   http://breakingmalware.com/vulnerabilities/one-bit-rule-bypassing-windows-10-protections-using-single-bit/
+ *   https://technet.microsoft.com/en-us/library/security/ms15-010.aspx
+ */

```

Comment:
To be picky, can we use the Doxygen style comments like we do in Meterpreter, pretty please? :)
```
/*~
 * @brief MS15-010 / CVE-2015-0057 win32k!xxxEnableWndSBArrows Use After Free
 * @author Udi Yavo (Vulnerability discover and details)
 * @author Spencer McIntyre (Exploit)
 * @remark Details can be found at http://breakingmalware.com/vulnerabilities/one-bit-rule-bypassing-windows-10-protections-using-single-bit/
 *         and https://technet.microsoft.com/en-us/library/security/ms15-010.aspx
 */
```